### DISCUSSION TITLE
CWTools download link out of date

### DISCUSSION THREAD

**Discussion by zach-hill** [bcssov/IronyModManager#327] [General]
The documentation for building the project locally includes a download link to `CWTools.Irony-Private.0.4.0-alpha4` but the project expects `alpha8`

Edit:

Oops, didn't mean to submit as a bug, but doesn't appear I can change it.

**Comment by bcssov** [comment]
This should probably be a discussion, it's not very often people ask me for this which is why I don't even bother anymore.

[CWTools.Irony-Private.0.4.0-alpha8.zip](https://github.com/bcssov/IronyModManager/files/7798143/CWTools.Irony-Private.0.4.0-alpha8.zip)
